components/IndicatorStat/IndicatorStatDataHandler.tsx

```diff
@@ -8,20 +8,44 @@ interface DataHandlerProps {
   dataLoading?: boolean;
   valueOptions?: ValueOptions;
   note?: ContentNote;
+  decimalCount?: number;
 }
 
-const IndicatorStatDataHandler: FunctionComponent<DataHandlerProps> = ({ data, dataLoading, ...props }) => {
+const IndicatorStatDataHandler: FunctionComponent<DataHandlerProps> = ({
+  data,
+  dataLoading,
+  decimalCount,
+  ...props
+}) => {
   if (!dataLoading && data) {
     if (data.length === 1) {
-      return <IndicatorStatDataViewer value={getIndicatorValue(data[0].data, props.valueOptions)} note={props.note} />;
+      return (
+        <IndicatorStatDataViewer
+          value={getIndicatorValue(
+            data[0].data,
+            props.valueOptions,
+            props.valueOptions?.suffix?.indexOf('th') === 0 ? 0 : decimalCount
```

Comment:
The errors this will raise ... cos things are optional for a reason. That aside, won't be needed if decimalCount is part of valueOptions.

Reply:
Changed to use if statements